core/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "libp2p-core"
 edition = "2021"
-rust-version = "1.56.1"
+rust-version = "1.60.0"
```

Comment:
Why are we changing the rust-version on some packets to `1.60.0` and on others to `1.62.0`?

Reply:
Some only require 1.60, others require 1.62 :)